jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -64,7 +66,7 @@
     }
 
     private Future<ImagesAndRegistryClient> baseImageAndRegistryClient = failedFuture();
-    private Future<List<Future<PreparedLayer>>> baseImageLayers = failedFuture();
+    private Future<Map<Image, List<Future<PreparedLayer>>>> baseImageLayers = failedFuture();
```

Comment:
What is the best name for this map object that maps Image - baseImageLayers ? I was thinking of ` imageLayers` or ` baseImagesAndLayers `

Reply:
`baseImagesAndLayers` sounds good to me.

Reply:
After addressing the comments, I think you can take this out of the draft status.

Reply:
Sounds Good